tests/conftest.py

```
@@ -5,216 +5,141 @@
                 str(app_path)
             ],
-            stdout=subprocess.PIPE,
-            stderr=subprocess.PIPE,
+            stdout=subprocess.DEVNULL, # หรือใช้ sys.stdout ถ้าต้องการดู log
+            stderr=subprocess.STDOUT,
             env=env,
             cwd=str(project_root),
             text=True
         )
         print(f"✅ Subprocess started (PID: {process.pid})")
     except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)